portfolio.css

```diff
@@ -2108,4 +1064,805 @@ input[type=email] {
+  }
+
+  .header {
+    width: 2063px;
+    height: 72px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .menu {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 8px;
+    padding: 0;
+  }
+
+  .menu-items1 {
+    list-style-type: none;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 87px;
+    height: 40px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    text-align: center;
+    color: #344563;
+  }
+
+  .menu-items2 {
+    list-style-type: none;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 69px;
+    height: 40px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    text-align: center;
+    color: #344563;
+  }
+
+  .menu-items3 {
+    list-style-type: none;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 83px;
+    height: 40px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    text-align: center;
+    color: #344563;
+  }
+
+  .desktop-headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 160px 0;
+    gap: 24px;
+    width: 2063px;
+  }
+
+  .desktop-headline-first-text {
+    width: 920px;
+    height: 114px;
+  }
+
+  .desktop-headline-heading {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    text-align: center;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+  }
+
+  .desktop-headline-heading span {
+    color: rgb(92, 129, 92);
+  }
+
+  .desktop-headline-supporting-text {
+    width: 920px;
+    height: 96px;
+  }
+
+  .desktop-headline-secondary-text {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .desktop-social-media-icons {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    width: 200px;
+    height: 40px;
+    gap: 8px;
+  }
+
+  .desktop-social-media-icons > li {
+    list-style-type: none;
+    align-items: flex-start;
+    padding: 8px;
+    width: 40px;
+    height: 40px;
+    border-radius: 8px;
+  }
+
+  .desktop-normal-button {
+    width: 920px;
+    height: 114px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .btn2 {
+    width: 40px;
+    height: 40px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 32px;
+  }
+
+  .btn2:disabled {
+    padding: 0;
+    width: 40px;
+    height: 40px;
+    cursor: none;
+  }
+
+  .desktop-work {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 300px 0;
+    gap: 48px;
+    width: 2063px;
```

Comment:
- [ ] Kindly note that the `_**Work Section**_` section for the desktop view is currently not centered. This is attributed to the width you specified for the section ( width: 2063px; ). Kindly remove the width and the section will be centralized just as the previous reviewer suggested 😄 

![image](https://user-images.githubusercontent.com/63473584/187044519-184dd2f7-1d6a-4f72-b6c9-025d02e2f3eb.png)